debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java

```diff
@@ -287,9 +287,6 @@ public static ConfigDef configDef() {
                 }
             }
         }
-        if (!eventTypeMappingProvided) {
```

Comment:
Also the `ENVELOPE_EVENT_TYPE` constant and this variable `eventTypeMappingProvided` should be removed. With this change, there'll be nothing special about event type any longer.